### DISCUSSION TITLE
General

### DISCUSSION THREAD

**Discussion by Lasha-Ilashvili** [Lasha-Ilashvili/KotlinInterpreter_FOP-Capstone#1] [General]
ამ მესიჯით დავიწყებთ GitHub-ზე კომუნიკაციას

**Comment by Lasha-Ilashvili** [comment]
INIT_PROJECT ბრენჩს დავფუშავ და დავმერჯავ დეველოპზე, აქ იქნება ჯავას პირველადი პროექტი და დამირევიუეთ

**Comment by MariamRomelashvili** [comment]
რუბის ფაილების დაწერა დავიწყოთ, იმპლემენტაცია გავწეროთ

**Comment by Lasha-Ilashvili** [comment]
ახლა ვფიქრობ უბრალოდ init რაც არის პროექტის ექ დავფუშო მეინზე, დროებით ნორმალური იქნება

**Comment by SamoleChokuri** [comment]
ხომ არ მოვხსნათ ხელით შეყვანა სკანერში ჯობს ეგრე მგონი ,ზედმეტი სინტაქსური შეცდომა რომ არ დაუშვან შეყვნისას ,ან მაშინ ცალკე გავწეროთ რაც არ შეუძლიათ მარა ბევრი გამოვა ეგრე

**Reply by Lasha-Ilashvili** [reply]
გასწერია ყველაფერი რასაც ვასაფორთებთ, შემდეგ შესამოწმებელია მაგას გვაწოდებს თუ არა იუზერი რაც გვინდა. და ერორს დავარტყამთ მაშინ. ოღონდ ეგ კითხვის ლოგიკაში არ უნდა იყოს, ცალკე უნდა გვქონდეს
